seth-shaw-unlv/controlled_access_terms
# Move RDF configs to optional.

**GitHub Issue**: #119 

## Other Relevant 
* General discussion of Recipes


# What does this Pull Request do?

Makes it possible to install Defaults without RDF.

# What's new?

* RDF removed from Defaults' requirements in info.yml
* RDF-related configs moved to optional folder
* RDF added as a suggest in composer.json.

* Does this change require documentation to be updated?  No
* Does this change add any new dependencies?  NO :) 
* Does this change require any other modifications to be made to the repository
 (i.e. Regeneration activity, etc.)? No
* Could this change impact execution of existing code? No

# How should this be tested?

* Launch a fresh drupal, e.g with [DDEV](https://ddev.readthedocs.io/en/latest/users/quickstart/#drupal).
* `composer require drupal/controlled_access_terms`
* `drush en controlled_access_terms_defaults` (fails, RDF is required)
* With this PR, it should install without RDF.
* Also, if you had RDF installed already, it should install the RDF configs.

# Additional Notes:
Any additional information that you think would be helpful when reviewing this
 PR.

# Interested parties
Tag (@ mention) interested parties or, if unsure, @Islandora/committers
